lms/services/lti_grading/_v13.py

```diff
@@ -22,7 +23,7 @@ def read_result(self, grading_id):
         try:
             response = self._ltia_service.request(
                 "GET",
-                self.grading_url + "/results",
+                self._service_url(self.grading_url, "/results"),
                 scopes=self.LTIA_SCOPES,
                 params={"user_id": grading_id},
```

Comment:
Note to self: I've given this a quick check and requests will take care of merging the params here into those on the provided URL